src/github/types.ts

```diff
@@ -43,15 +43,15 @@ export interface IIssues {
    * Returns the node_id for all the issues available in the repository
    * @param includeClosed exclude issues which are closed from the data agregation.
    */
-  getAllIssues(excludeClosed: boolean): Promise<Issue[]>;
+  getAllIssues(excludeClosed: boolean, labels?: string[]): Promise<Issue[]>;
 }
 
 export interface ILogger {
   info(message: string): void;
   warning(message: string | Error): void;
   error(message: string | Error): void;
   /** Only posts messages if the action is ran in debug mode */
-  debug(message: string): void;
+  debug(message: string, data?: Record<string, unknown>): void;
```

Comment:
I'm not sure about this, you just changed it to match your use-case here but it's not consistent with the rest of the log levels.

Consider either:
- Changing the interface of other loglevels (info, warning, etc) so it is consistent
- Reverting this change, handling the `data` outside of the logger, and proposing a change to the logger in a separate PR

Or maybe I'm overthinking it since this logger is only used in this package.

Reply:
Yeah, it is inconsistent. Reverting.